samples/boards/reel_board/mesh_badge/README.rst

```diff
@@ -0,0 +1,43 @@
+.. _mesh_badge:
+
+Mesh Badge
+##########
+
+Overview
+********
+
+A sample app for the reel board that showcases Bluetooth Mesh
+
+The app starts off as a regular Bluetooth GATT peripheral application. It's
+recommended to install the "nRF Connect" app on your phone (available both for
+Android and iOS) to access the service that the app exposes.
```

Comment:
Can you do this board setup with a program running on your laptop (or only with your phone)?

Reply:
yes, I added a note about that, but I'm not planning to provide detailed instructions for every possible OS, Bluetooth stack and GATT client app :)